tools/bliss-to-ogg-zip.py

```diff
@@ -255,6 +255,8 @@ def main():
   arg_parser.add_argument("bliss_filename")
   arg_parser.add_argument("--subset_segment_file")
   arg_parser.add_argument("--no_ogg", help="skip generating ogg files", action="store_true")
+  arg_parser.add_argument("--no_conversion", help="skip ffmpeg call, assume audio is correct already",
+                          action="store_true")
```

Comment:
Same formatting comment as below.

Reply:
Note, for the reasoning of this: Now when you change the name of `arg_parser`, this will mess up the formatting. And if you fix the formatting afterwards, the commit diff becomes more convoluted.
Also, this style can lead to cases where stuff is extremely right indented, which is bad.
I think you can find some articles which go into more detail. Maybe it's also a matter of taste in the end, but we should stay consistent in our code style.

Reply:
I found the option in PyCharm which enables the correct automatic line breaking when exceeding 120 characters, I will add this to the Wiki page.